### DISCUSSION TITLE
More info about is_vehicle_active

### DISCUSSION THREAD

**Discussion by FilipDem** [bimmerconnected/bimmer_connected#424] [Q&A]
Hi,
I would like to know when the car is driving or not...
I am using the vehicle status `is_vehicle_active`, referring to the code below.
You can also find an extract of the `bimmerconnected status`. In this status you find back 2 properties: "is_vehicle_active" and "inMotion" (see extract below). The function in the library returns the property "inMotion" (see code below).

What is the difference between both properties "is_vehicle_active" and "inMotion"?

I get the "bimmerconnected status" while the car is driving, however the status return "false" for both properties "is_vehicle_active" and "inMotion" (the extract of the status below is a real example while the car was driving). 
Has somebody the same experience? Somebody requested the status of the car while driving?

Thanks for feedback!

```
    def is_vehicle_active(self) -> bool:
        """Check if the vehicle is active/moving.
        If the vehicle was active/moving at the time of the last status update, current position is not available.
        """
        return self.properties['inMotion']

```

```
    "status": {
        "status": {
            "lastUpdatedAt": "2022-02-28T08:19:23Z",
        "properties": {
            "lastUpdatedAt": "2022-02-28T08:19:23Z",
            "inMotion": false,
        },
        "is_vehicle_active": false,
    },

```

**Comment by rikroe** [comment]
`properties.inMotion` is the value returned by the MyBMW API.
`is_vehicle_active` is our parsed value that we parse just 1-to-1 as we had that originally based on another value from the old API.

To my knowledge, only the newest cars (current 3 series, i4, iX, iX3,...) actually support the `inMotion` attribute to be updated automatically.

However I was not able to pinpoint any specified required head units or ilevels yet.
